lib/internal/process/next_tick.js

```diff
@@ -151,7 +145,11 @@ function setupNextTick() {
         args[i - 1] = arguments[i];
     }
 
-    nextTickQueue.push(new TickObject(callback, args));
+    nextTickQueue.push({
+      callback: callback,
```

Comment:
it doesn't appear to make a difference. Just ran again using shorthand and the numbers are more or less the same. :]